src/univariate/continuous/exponential.jl

```diff
@@ -60,6 +60,17 @@ kurtosis(::Exponential{T}) where {T} = T(6)
 
 entropy(d::Exponential{T}) where {T} = one(T) + log(d.θ)
 
+"""
+    kldivergence(p::Exponential, q::Exponential)
+
+See [KL Exponential](https://en.wikipedia.org/wiki/Exponential_distribution#Kullback%E2%80%93Leibler_divergence)
+"""
+function kldivergence(p::Exponential, q::Exponential)
+    λp = scale(p)
+    λq = scale(q)
+    return log(λp) - log(λq) + λq / λp - 1
```

Comment:
`logmxp1` return `log(x) - x +1` so one would still need to compute the `log` additionally to correct for it

Reply:
Also : isn't `log(x) - log(y)` more stable than `log(x / y)` ?

Reply:
No, not necessarily since due to cancellation the result of `log(x) - log(y)` can be much less precise.

> logmxp1 return log(x) - x +1 so one would still need to compute the log additionally to correct for it

No, if one would compute `logmxp1(lp_over_lq) = log(lp_over_lq) - lp_over_lq + 1` one would just have to subtract 2. But the precision wouldn't be as optimal since `logmxp1` is optimized for the `+1` addition.